action.yml

```
@@ -81,6 +81,13 @@ runs:
     - name: 'Install gdUnit4 plugin - ${{ inputs.version }}'
       shell: bash
       run: |
+          if ${{ inputs.version == 'installed' }}; then
+            echo -e "\e[33m Info: Using the installed GdUnit4 plugin. \e[0m"
+            gdunit_version=$(grep -oP '^version="\K[^"]+' ./addons/gdUnit4/plugin.cfg)
```

```suggestion
            gdunit_version="v$(grep -oP '^version="\K[^"]+' ./addons/gdUnit4/plugin.cfg)"
```
The version must start with 'v' e.g. 'v4.2.0'